dll/win32/shell32/precomp.h

```diff
@@ -181,7 +181,7 @@ Shell_DisplayNameOf(
     _Out_ LPWSTR pszBuf,
     _In_ UINT cchBuf);
 
-HRESULT SHBindToObject(
+EXTERN_C HRESULT SHBindToObject(
```

Comment:
```suggestion
EXTERN_C
HRESULT SHBindToObject(
```

Reply:
The formatting in this file is all over the place, even SHBindToObject vs SHBindToObjectEx are different!